Cargo.toml

```diff
@@ -62,8 +62,8 @@ x11-dl = "2"
 winpty = { path = "./winpty" }
 mio-named-pipes = "0.1"
 miow = "0.3"
-winapi = { version = "0.3.5", features = ["impl-default", "winuser", "synchapi", "roerrorapi", "winerror", "wincon"]}
 dunce = "1.0"
+winapi = { version = "0.3.7", features = ["impl-default", "winuser", "synchapi", "roerrorapi", "winerror", "wincon"]}
```

Comment:
Do you need wincontypes here?

Reply:
It's not needed as it's a sub-feature of the wincon feature. But I think we might aswell be explicit and add it.